src/main/java/com/aws/greengrass/secretmanager/LocalStoreMap.java

```diff
@@ -71,17 +71,16 @@ private boolean validUpdate(Node node, String key) {
 
     protected Crypter getCrypter() throws SecretCryptoException {
         synchronized (crypterLockObject) {
-            if (crypter.get() == null) {
-                try {
-                    loadCrypter();
-                } catch (Exception e) {
-                    throw new SecretCryptoException("Unable to load crypter", e);
-                }
+            try {
+                loadCrypter();
```

Comment:
Why are we creating a new crypter everytime we need to encrypt or decrypt a secret?